### DISCUSSION TITLE
90B74093-416A-88FC-6FF6-668B47898472

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#149] [General]
# Extreme TNOs

Adds various ETNOs

https://celestia.mobi/addon?item=90B74093-416A-88FC-6FF6-668B47898472&lang=it

**Comment by massimofedrigo** [comment]
Wonderful but 2015 TG387/TheGoblin is bugged.

**Comment by hachibi108** [comment]
Can you add 2015 RR245 and ǂKá̦gára and his moon !Hãunu?
